### ISSUE TITLE
...the website is down

### ISSUE THREAD

**Issue by d3nwah** [Lameguy64/PSn00bSDK#12]
And with it I find it hard to find the precompiled GCC 7.4.0 toolkit 🙁

**Comment by Lameguy64** [comment]
Should be up a long while ago. Make sure you use the new lameguy64.net domain.

**Comment by sauntheninja2** [comment]
The new domain also seems to be dead it redirects to irix Network

**Comment by Lameguy64** [comment]
Not really, something about address redirection got screwed up when my host implemented a vulnerability fix on their servers and I've since reported the problem. The domain is still up until January 29 of next year and I intend to renew it after new year.
